src/sptps.h

```diff
@@ -39,6 +40,14 @@
 #define SPTPS_SIG 3           // Waiting for a SIGnature record
 #define SPTPS_ACK 4           // Waiting for an ACKnowledgement record
 
+// max transmission unit size
+// 1500 bytes usable space for the ethernet frame or 9000 for the jumbograms
+// - 20 bytes IPv4-Header
+// -  8 bytes UDP-Header
+// - 19 to 21 bytes encryption (sptps.c send_record_priv / send_record_priv_datagram)
+#define SPTPS_MTU (PAYLOAD_MTU - 47)
```

Comment:
IPv6 still needs to be handled, right, but for IPv4 I'm pretty sure meshlink doesn't set additional options, they are rarely used anyway